Formula/root.rb

```diff
@@ -85,16 +93,25 @@ def install
       -Dxrootd=ON
     ]
 
+    # This will become -DCMAKE_CXX_STANDARD=14 (or 17) in ROOT 6.18
```

Comment:
If this is making a change to how C++ support is being compiled, is it worth bumping the revision for it?

Reply:
We can. It now matches the maximum allowed on your system. I believe this mostly affects people using the C++ interpreter; it will now report a higher C++ value if asked. It also allows ROOT to use several features in the stdlib that otherwise would use a custom backport, but that shouldn't affect users.

Do I just add a `_0` to the version?

Reply:
Just add `revision 1` underneath the version!

Reply:
Done!

Reply:
Are there any packages that depend on ROOT in Homebrew (for instance, rivet, geant4-vmc, etc)? If yes, then you might need to make sure they depend on ROOT with the same C++ standard, or they will likely fail to compile.